### ISSUE TITLE
Ethernet link state broken

### ISSUE THREAD

**Issue by flohoff** [milkv-duo/duo-buildroot-sdk#105]
With the milkv-duos-sd-v1.1.1-2024-0528.img the Ethernet link State on the Milk-DUO-S is detected the wrong way around. Without a cable attached the LEDs lit up and the network interface says

```
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP qlen 1000
    link/ether a2:0b:5d:3b:89:ff brd ff:ff:ff:ff:ff:ff
    inet 169.254.129.156/16 brd 169.254.255.255 scope global noprefixroute eth0
       valid_lft forever preferred_lft forever
```

When attaching a cable the LED turn off and the interface says:

```
2: eth0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc mq state DOWN qlen 1000
    link/ether a2:0b:5d:3b:89:ff brd ff:ff:ff:ff:ff:ff
```

I guess the driver for the Ethernet needs some love.

**Comment by flohoff** [comment]
Okay - some failure here - that we didnt get a link on cable connect was a cabling issue. Still we continously see a link even if there is no cable connected.

**Comment by sverdlin** [comment]
Indeed, I've reported the same problem against the Sophgo documentation some time ago (because without documentation further debugging is impoissible): https://github.com/sophgo/sophgo-doc/issues/170

Unfortunately the vendor shows no interest in debugging this.
